draft-ietf-rats-architecture.md

```diff
@@ -842,40 +842,47 @@ a different format.
+
+There are two common approaches for determining the epoch of an Evidence or Attestation Result.
+
+<!-- Explicit Timekeeping using Synchronized Clocks -->
+
+The first approach is to rely on synchronized and trustworthy clocks, and include a signed timestamp (see {{?I-D.birkholz-rats-tuda}}) along with the Claims in the Evidence or Attestation Result.
+Timestamps can also be added on a per-Claim basis if wanted, to allow distinguishing the time of creation of Evidence from the time that a specific Claim value was generated.
+Clocks trustworthiness typically requires additional Claims about the signer's time synchronization mechanism.
+
+<!-- Implicit Timekeeping using Nonces -->
+
+A second approach places the onus of timekeeping solely on the appraising entity (e.g., the Verifier for Evidence, or the Relying Party for an Attestation Result).
+This might be an appropriate choice in case the Attester does not have a reliable clock or time synchronisation is otherwise impaired.
+In this approach, a non-predictable nonce is generated by the appraising entity, and the nonce is then signed and included along with the Claims in the Evidence or Attestation Result.
+After checking the sent and received nonces are the same, the appraising entity knows that the Claims were signed after the nonce was generated, effectively locking its clock to the produced payload.
+This allows associating a "rough" epoch to the Evidence or Attestation Result.
+In this case the epoch is said to be rough because:
+
+* it is always bound to the set of Claims as a whole, as opposed to having Claim being separately annotated, and
+* it is not possible to distinguish between time of collection from the time of creation of the Claims set.
+
+The two basic mechanisms presented above can be extended and combined in more advanced schemes.
+For example, if the clocks are trustworthy but not synchronized, first a nonce-based exchange may be used to determine the (rough) time offset between the involved peers, followed by any number of timestamp based exchanges.
+In another scenario where a broadcast channel is shared by all the RATS entities (Attesters, Verifiers and Relying Parties), the nonce-based approach may be used to lock all parties to the same timeline without requiring synchronized clocks by having a central entity emit nonces at regular intervals and have the "current" nonce included in the produced Evidence or Attestation Result.
+
+Whatever the approach, it is important to note that the actual values in Claims might have been generated long before the Claims are signed.
+If so, it is the signer's responsibility to ensure that the values are still correct when they are signed.
+For example, values might have been generated at boot, and then used in Claims as long as the signer can guarantee that they cannot have changed since boot.
```

Comment:
This is not an example imo. "For example, values generated at boot time might have been saved to secure storage until network connectivity is established to the remote Verifier and a nonce is obtained."

Reply:
fixed in c215569